src/material-experimental/mdc-button/fab.scss

```diff
@@ -1,17 +1,28 @@
 @import '@material/fab/mixins';
+@import '@material/button/variables';
+@import '@material/theme/variables';
 @import '../mdc-helpers/mdc-helpers';
 
 @include mdc-fab-without-ripple($query: $mat-base-styles-query);
 
 $_mat-button-ripple-opacity: 0.1;
 
-.mat-mdc-button {
+.mdc-fab {
   // The ripple element is created from the RippleRenderer rather than the MDC ripple, so it is
   // necessary to provide the color and opacity styling manually.
   .mat-ripple-element {
     opacity: $_mat-button-ripple-opacity;
     background-color: currentColor;
   }
+
+  // Spec does not define disabled FAB buttons but Angular Material does support it.
+  &:disabled {
+    background-color: rgba(mdc-theme-prop-value(on-surface), 0.12);
+    color: $mdc-button-disabled-ink-color;
```

Comment:
does this work in both light & dark theme?

Reply:
Looks like our theming support is not yet implemented for the button (does not support accent/primary). Will revisit this PR when that is completed

Reply:
LGTM - Verified with https://github.com/angular/components/pull/16782, looks good